packages/components/src/constants/preferences.ts

```diff
@@ -0,0 +1,226 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Created a `constants` subfolder and moved all preferences over since these were needed by the components

Reply:
I'd rather not bring all of these constants over 1-1. Is there some subset this component needs?

Reply:
Ya let's keep these local to where they are needed rather than in a constants file. I would like to kill that pattern.

Reply:
What about constant that are needed across multiple components? The constants I needed here were used in these components but also in other places in the codebase. Having duplicates of the constants seems more problematic, doesn't it?